client/my-sites/plans/jetpack-plans/product-grid/style.scss

```diff
@@ -107,13 +107,56 @@
 }
 
 .product-grid__pricing-banner {
-	height: 54px;
+	height: 120px;
+
+	margin-bottom: 20px;
+
+	@include break-large {
+		height: 54px;
+
+		margin-bottom: 0;
+	}
 }
 
 .product-grid__more.is-detached {
 	margin-top: 48px;
 }
 
+.product-grid__more-container {
+	background-color: var( --color-surface );
+
+	margin: 24px 0;
+	padding: 30px 15px 26px;
+	border-radius: 8px; /* stylelint-disable-line scales/radii */
```

Comment:
A lot of the styles you see are copied from a very similar component. The design team wanted it to look the same but the component I copied from was not nearly reusable enough to just use that instead, it had a lot of very specific conditions within it for what it was being used. It is called `SupportCard`. Anyway, I should have looked closer at the code instead of just copying it and calling it good :)